cfp/gophercon-2024-should-I-use-IoC.md

```diff
@@ -0,0 +1,36 @@
+# Title
+Should I use Inversion of Control Containers in Go?
+
+# Abstract
+Inversion of Control (IoC) Containers, evil or just misunderstood? Lets explore this tool at a high level so you can decide for yourself. We'll cover what an IoC Container is, when to use it, when not to use it, and some things to watch out for.
+
+# Format
+I chose the 25-Minute track vs. the more in-depth 45 minute track because I believe the community would benefit more from a high-level overview of IoC Containers vs. a deep dive.
+
+25-Minute Talk - Should present big ideas of general interest to the entire attendee base. The audience will be comprised of Go programmers of all experience levels.
+
+# Outline
+* (5 Minutes) Introduction
+    * Who am I and what's my background
+    * My experiences with the Go community's polarized views on IoC Containers
```

Comment:
Add anecdote about first exposure to IoC Containers in Doom source code.

Reply:
This appears to be a false memory and I can't find the information to back this up 😆 
Will reflect on that and see what I was thinking of.